gateway/webrtc/client.js

```diff
@@ -247,59 +258,44 @@ function handleVideo(myStream) {
     //when a remote user adds stream to the peer connection, we display it
     yourConn.ontrack = function(e) {
         console.log("Entering ontrack", e);
-        
-        callBtn.style.display='none';
-        callToUsernameInput.style.display='none';
-        hangUpBtn.style.display='inline';
-        overlay.style.visibility='visible';
-        overlay.innerText='Connected to '+connectedUser;
-
-        if (true == answerReceived) { 
-	   if ( e.track.kind==="video") {
-		console.log("Adding video stream");
-       		remoteVideo.srcObject = e.streams[0];
-	   }
-	   console.log("Exiting ontrack"); 
-	   return; 
-	}
-
-        if (connectedUser !== undefined && connectedUser.length > 0 )  { 
-	   if ( e.track.kind==="video") {
+        showInCall();
+        if ( e.track.kind==="video") {
+            console.log("Adding video stream");
                 remoteVideo.srcObject = e.streams[0];
-                yourConn.createOffer().then(function(offer) {
-                        console.log("Creating offer : ", offer);
-                        send({
-                            type: "offer",
-                            offer: offer
-                        });
-
-                        yourConn.setLocalDescription(offer);
-                        
-
-                    }).catch(function(error) {
-                        console.log("Error when creating an offer", error);
-                    });
-       } 
-	} 
+        }
         console.log("Exiting ontrack");
     };
 
     yourConn.addStream(stream);
-    //stream.getTracks().forEach(track => yourConn.addTrack(track, stream));
+
     // Setup ice handling
     yourConn.onicecandidate = function(event) {
         console.log("Entering onicecandidate", event);
+        
         if (event.candidate) {
-            // comment in if you want to force relay, this demo currently requires
-            if (event.candidate.candidate.indexOf("relay") > 0) {
+            if (event.candidate.candidate.indexOf("host") < 0) {
```

Comment:
Please add a constant to force going through relay and default to true (configuring this value via the UI would be a plus)